task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +115,17 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return function() {
+        for (let i = 0; i <= attempts; i++){
+            try {
+                return func();
+            } catch (error) {}
+
+            if(i == attempts){
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"